packages/block-editor/src/components/block-preview/index.js

```diff
@@ -28,12 +28,22 @@ function BlockPreview( props ) {
 	);
 }
 
-export function BlockPreviewContent( { name, attributes, innerBlocks, settings } ) {
-	const block = createBlock( name, attributes, innerBlocks );
+export function BlockPreviewContent( { blocks, settings } ) {
+	if ( ! blocks ) {
+		return null;
+	}
+
+	const theBlocks = Array.isArray( blocks ) ? blocks : [ blocks ];
+
+	// Create new "clone" Blocks to show in the preview
+	const blocksToPreview = theBlocks.map( ( { name, attributes, innerBlocks } ) => {
```

Comment:
Why do we need to clone the blocks? Can't we just rely on the caller? 
Also, if we do, there's a `cloneBlock` function somewhere I believe.

Reply:
I've re-tested and I don't think it is required. I feel like I introduced it cos there was an error. But looking more closely I think the result is the same either way.